dat/missions/zalek/neburesearch/neburesearch_00.lua

```diff
@@ -0,0 +1,221 @@
+--[[
+   
+   Mission: Novice Nebula Research
+   
+   Description: A Za'lek student asks the player to help him with his reasearch. The player has to visit Doeston and Arandon.
+   Some minor complications on the way have to be expected.
+   
+   Difficulty: Easy
+
+]]--
+
+include "dat/scripts/numstring.lua"
+
+
+t_sys = {}
+t_sys[1] = "Doeston"
+t_sys[2] = "Arandon"
+
+-- localization stuff, translators would work here
+bar_desc = _("You see a young scientist talking with some pilots, apparently without success.")
+mtitle = {}
+mtitle[1] = _("Novice Nebula Research")
+misn_reward = _("%s credits and the gratitude of a student")
```

Comment:
You should replace `%s credits` with `%s` and use the `creditstring` function found in numstring.lua. This will make sure that it can be translated properly.

🕵️